Assets/Scripts/SoundSystem/AudioDataSO.cs

```diff
@@ -9,12 +9,17 @@ public class AudioDataSO : ScriptableObject
 
 	private GameObject gameObject;
 	private AudioSource audioSource;
+	private const float VolumeMaxVal= 1f;
+	//set default max value to 1f
+	[Range(0f, VolumeMaxVal)]
+	public float volSlide;
 
 	public void Play()
 	{
 		if(gameObject != null && audioSource != null)
 		{
 			audioSource.Stop();
+			audioSource.volume = volSlide;
```

Comment:
You need to add this when the audio firsts gets set, otherwise it will only set the volume when the audio is played a second time